spec/lib/api/v3/work_packages/schema/work_package_schema_spec.rb

```diff
@@ -108,24 +108,53 @@
     end
 
     describe '#assignable_priorities' do
-      let(:result) { double }
+      let(:active_priority) { FactoryGirl.build(:priority, active: true) }
+      let(:inactive_priority) { FactoryGirl.build(:priority, active: false) }
 
-      it 'calls through to the work package' do
-        expect(work_package).to receive(:assignable_priorities).and_return(result)
-        expect(subject.assignable_priorities).to eql(result)
+      before do
+        active_priority.save!
+        inactive_priority.save!
+      end
+
+      it 'returns only active priorities' do
+        expect(subject.assignable_priorities.size).to be >= 1
+        subject.assignable_priorities.each do |priority|
+          expect(priority.active).to be_truthy
+        end
+      end
+    end
+
+    describe '#assignable_categories' do
+      let(:category) { double('category') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.